website/content/docs/upgrading/version-guides/0.10.0.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+page_title: Version 0.10.0 - Upgrade Guide
+description: |-
+  This upgrade guide describes any considerations for upgrading to Waypoint 0.10.0.
+---
+
+# Upgrading to Waypoint 0.10.0 from 0.9.x
+
+Waypoint 0.9.x can be upgraded to Waypoint 0.10.0 using a [standard upgrade](/docs/upgrading#standard-upgrade).
+There are no breaking changes in the protocol or `waypoint.hcl` configuration file. All
+Waypoint components (CLI, entrypoint, and server) are forwards and backwards
+compatible, but we encourage users to have all components on the same version.
+
+# Breaking Changes
```

Comment:
Remove "Breaking Changes" if there aren't any? If it should stay, change to `## Breaking Changes` so it's a child of the one top-level `# Upgrading to Waypoint 0.10.0 from 0.9.x`?

Otherwise, LGTM!

Reply:
so removed!